src/app/dropdown/dropdown.component.css

```diff
@@ -131,6 +131,24 @@ hr {
     padding-top: 10px;
 }
 
+@media screen and (max-width: 2670px) {
+    .dropdown-menu-box {
+        margin-left: -80%;
+    }
+}
+
```

Comment:
@harshit98 Simply change to min-width 817 px (or from whichever size you want the change to be applied). This is redundant.